app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileState.kt

```diff
@@ -8,7 +10,55 @@ data class UserProfileState(
     val fullName: String,
     val userName: String,
     val teamName: String, //maybe teamId is better here
-    val otherAccounts: List<OtherAccount>
+    val otherAccounts: List<OtherAccount>,
+    val dialogState: DialogState = DialogState.None
 )
 
 data class OtherAccount(val id: String, val avatarUrl: String, val fullName: String, val teamName: String? = null)
+
+sealed class DialogState {
```

Comment:
Maybe better if renamed to
```suggestion
sealed class StatusChangeDialogState {
```
Otherwise it indicates this class could be applied to any user dialog